pkg/hubble/relay/defaults/defaults.go

```diff
@@ -45,4 +47,8 @@ var (
 	// ListenAddress is the address on which the Hubble Relay server listens
 	// for incoming gRPC requests.
 	ListenAddress = fmt.Sprintf(":%d", hubbledefaults.RelayPort)
+
+	// HealthListenAddress is the address on which the Hubble Relay gRPC health
+	// server listens on
+	HealthListenAddress = fmt.Sprintf(":%d", 4222)
```

Comment:
Shouldn't this be `localhost:4222`?

Reply:
I thought so too, but no because the built-in gRPC probes don't originate from the container itself (which makes sense in hindsight). So we can't just listen on localhost.

That's not ideal, but I also don't really see an issue with answering health probes in plain text.